src/tools/ipv4-range-expander/result-row.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <tr>
+    <td>
+      <n-text strong>{{ label }}</n-text>
+    </td>
+    <td :data-test-id="testId + '.old'"><span-copyable :value="oldValue" class="monospace" /></td>
+    <td :data-test-id="testId + '.new'">
+      <span-copyable :value="newValue"></span-copyable>
+    </td>
+  </tr>
+</template>
+
+<script setup lang="ts">
+import SpanCopyable from '@/components/SpanCopyable.vue';
+import { paramCase } from 'change-case';
```

Comment:
[nit]
I'd prefer using `_.kebabCase` from lodash
They are more chance that I replace `change-case` than `lodash` 😅